package.json

```diff
@@ -57,6 +58,7 @@
     "prismjs": "^1.5.1",
     "sass-loader": "^3.2.3",
     "style-loader": "^0.13.1",
+    "uglifyjs-webpack-plugin": "^1.0.0-beta.1",
```

Comment:
Is there a reason to specifically use this version instead of the latest version of 1.3 or 2.1.1?

[Version list](https://www.npmjs.com/package/uglifyjs-webpack-plugin)